plugins/chat/test/javascripts/acceptance/chat-flagging-test.js

```diff
@@ -53,7 +53,9 @@ acceptance("Discourse Chat - Flagging test", function (needs) {
     assert.notOk(exists(".chat-live-pane .chat-message .chat-message-flagged"));
     await triggerEvent(".chat-message-container", "mouseenter");
 
-    let moreButtons = selectKit(".chat-msgactions-hover .more-buttons");
+    let moreButtons = selectKit(
```

Comment:
I know you just renamed the element selector, but this could be a `const`:

```suggestion
    const moreButtons = selectKit(
```